src/modules/jwt/jwt_mod.c

```diff
@@ -172,18 +175,28 @@ static int jwt_fcache_add(str *key, str *kdata)
  *
  */
 static int ki_jwt_generate(sip_msg_t* msg, str *key, str *alg, str *claims)
+{
+	return ki_jwt_generate_hdrs(msg, key, alg, claims, NULL);
+}
+
```

Comment:
This function has to be moved after `ki_jwt_generate_hdrs()` in order to get it compiled.